sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/SpringAppDeployment.java

```diff
@@ -182,10 +194,10 @@ interface DefinitionStages {
 
             /**
              * Specifies the cpu number of the deployment.
-             * @param cpuCount the number of the cpu, 1 core can be represented by 1 or 1000m
+             * @param cpuCount the number of the cpu, can be 0.5, 1, 2, etc
              * @return the next stage of deployment definition
              */
-            T withCpu(String cpuCount);
+            T withCpu(double cpuCount);
```

Comment:
Good is easy to use, bad is service is not obligated to keep the same format within api-version (though not likely to change).